statsmodels/tsa/statespace/tools.py

```diff
@@ -56,10 +57,16 @@ def set_mode(compatibility=None):
     # Initialize the appropriate mode
     if not compatibility:
```

Comment:
was it determined that compatibility is going away soon?

Reply:
If this refers to the scipy version, then yes.
I think we should drop scipy < 0.16 right after the release of 0.9.

Reply:
or we can even drop more if needed.

Reply:
Should I hold off on further review until some upcoming set of edits?